gallery/src/styles/Nivo/NivoPieExample.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { pieData } from './data';
+
+export default function NivoPieExample() {
+  return (
+    <>
+      <div style={{ height: '400px' }}>
```

Comment:
Without this it seems like the chart grows indefinitely over time.  If there's a general rule that the chart be contained within an element with a definite height, we should spell that rule out.